.gitignore

```diff
@@ -0,0 +1,4 @@
+.idea
```

Comment:
In this file you can specify which files will be ignored by git and never added to the repository.